components/brave_wallet/browser/fil_tx_state_manager.h

```diff
@@ -8,8 +8,14 @@
 
 #include <memory>
 #include <string>
+#include <utility>
+#include <vector>
 
+#include "base/gtest_prod_util.h"
```

Comment:
not used
Please re-check all headers in this file.

Reply:
done